corefile.cfg

```diff
@@ -0,0 +1,22 @@
+. {
+    proxy . /etc/resolv.conf
+    prometheus :9053
+}
+
+google.com {
+    proxy . /etc/resolv.conf
+    prometheus :9053
+    cache 60
+}
+
+github.com {
+    proxy . /etc/resolv.conf
+    prometheus :9053
+    cache 60
+}
+
+segment.com {
+    proxy . /etc/resolv.conf
+    prometheus :9054
+    cache 60
+}
```

Comment:
This file is only here as an example, I'll remove it once the code is ready to be merged.

Reply:
I missing something here, I think. But when you register a metrics in prometheus, currently via the sync.Once, so say `.` registers the metrics. Why don't the other (say) proxies plugins re-use that one?